Documentation/DocuBlocks/Rest/Replication/get_api_wal_access_tail.md

```diff
@@ -6,6 +6,10 @@
 
 @RESTQUERYPARAMETERS
 
+@RESTQUERYPARAM{global,boolean,optional}
+If set to *true*, tails the WAL for all databases. If set to *false*, tails the 
+WAL for the selected database.
```

Comment:
I'm confused. In the preview I generated there is an option global, but with a completely different description?!
https://deploy-preview-297--zealous-morse-14392b.netlify.com/docs/3.6/http/replications-walaccess.html#tail-recent-server-operations

Reply:
Retrying: https://jenkins01.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/549/